### PR TITLE
Add license of qwery, cc #9248

### PR DESCRIPTION
**Pull request by NiniHsu**
https://github.com/ded/qwery/blob/master/LICENSE

### CODE REVIEW

File: ajax/libs/qwery/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "type": "git",
     "url": "https://github.com/ded/qwery.git"
   },
+  "license":"MIT",
```
Comment (kennynaoh) [inline ajax/libs/qwery/package.json]: @NiniHsu 
Please add blank after `:`
Thanks!

**Comment by kennynaoh** [comment]
Hello @NiniHsu 
Please tag me after you fix this PR.
Thanks!

**Comment by NiniHsu** [comment]
Hi, @kennynaoh 
I fixed it.
Please take a look.
Thank you.

**Comment by kennynaoh** [comment]
@NiniHsu 
Please use more meaningful branch name next time.
Thanks!

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**